website/pages/en/subgraphs/quick-start.mdx

```diff
@@ -82,11 +82,8 @@ For a detailed breakdown on how to write your subgraph, check out [Creating a Su
 
 ### 5. Deploy your subgraph
 
-Remember, deploying is not the same as publishing.
-
-When you deploy a subgraph, you push it to [Subgraph Studio](https://thegraph.com/studio/), where you can test, stage and review it.
-
-When you publish a subgraph, you are publishing it onchain to the decentralized network.
+> Remember, deploying is not the same as publishing.
+When you **deploy** a subgraph, you push it to [Subgraph Studio](https://thegraph.com/studio/), where you can test, stage and review it. A deployed subgraph's indexing is performed by the [Upgrade Indexer](https://thegraph.com/blog/upgrade-indexer/), which is a single Indexer owned and operated by Edge & Node, rather than by the many decentralized Indexers in The Graph Network. A **deployed** subgraph is free to use, rate-limited, not visible to the public, and meant to be used for development, staging, and testing purposes.
```

Comment:
```suggestion
> Remember, deploying is not the same as publishing.

When you **deploy** a subgraph, you push it to [Subgraph Studio](https://thegraph.com/studio/), where you can test, stage and review it. A deployed subgraph's indexing is performed by the [Upgrade Indexer](https://thegraph.com/blog/upgrade-indexer/), which is a single Indexer owned and operated by Edge & Node, rather than by the many decentralized Indexers in The Graph Network. A **deployed** subgraph is free to use, rate-limited, not visible to the public, and meant to be used for development, staging, and testing purposes.
```